### DISCUSSION TITLE
How can I change the styles created by taiga-ui only in a current component?

### DISCUSSION THREAD

**Discussion by BlackBearDN** [taiga-family/taiga-ui#2189] [Q&A] [answered]
Good afternoon! The project uses the taiga-ui library, and when using the accordion component, taiga adds its own internal tags with unnecessary paddings. With a simple replacement via css styles, they do not change in any way. Is it possible to replace styles in this particular component without resorting to changing styles in the library files itself?
<img width="416" alt="Снимок экрана 2022-07-24 в 16 54 18" src="https://user-images.githubusercontent.com/64292028/180650350-604b0bcd-98c0-435f-b73c-5b90c6d7b0b7.png">

**Comment by waterplea** [comment, marked as answer]
I'm afraid not at the moment. Please file an issue to improve `Accordion` customization. There's a flag input for padding of the titles but nothing about the content. And overall a flag is a lazy solution so we need to reconsider it :)
